app/settings.py

```diff
@@ -59,9 +59,10 @@ def utcoffset_or_fail(date_value, key):
 
 EQ_PUBLISHER_BACKEND = os.getenv("EQ_PUBLISHER_BACKEND")
 EQ_FULFILMENT_TOPIC_ID = os.getenv("EQ_FULFILMENT_TOPIC_ID", "eq-fulfilment-topic")
-EQ_SUBMISSION_CONFIRMATION_TOPIC_ID = os.getenv(
-    "EQ_SUBMISSION_CONFIRMATION_TOPIC_ID", "eq-submission-confirmation-topic"
+EQ_SUBMISSION_CONFIRMATION_QUEUE = os.getenv(
+    "EQ_SUBMISSION_CONFIRMATION_QUEUE", "eq-submission-confirmation"
 )
+EQ_SUBMISSION_CONFIRMATION_BACKEND = os.getenv("EQ_SUBMISSION_CONFIRMATION_BACKEND")
```

Comment:
This needs to be added to the Helm deployment yaml as it is currently failing due to not backend.

Reply:
Added this to the helm chart